### PR TITLE
Quiz app changes

### PR DESCRIPTION
**Pull request by Safirangi**
#### Changes introduced in the [Quiz-App](https://github.com/mudit023/templates/tree/main/Quiz-App) template

- [x] edit README.md file to add screenshots
- [x] add HTML comments
- [x] add CSS variables
- [x] add JavaScript comments

this fixes issue #112

### CODE REVIEW

**Comment by mudit023** [comment]
Hey, thanks for contributing. Can you add one gif of the template also?

File: Quiz-App/README.md [hunk 1]
```diff
@@ -23,6 +24,8 @@ Users should be able to:
 
 ### Screenshot
 
+![Question Interface Card](./question-interface.png)
+![Results Card](./result.png)
```
Comment (mudit023) [inline Quiz-App/README.md:28]: Please add a gif also.

Comment (Safirangi) [reply Quiz-App/README.md:28]: yes sure 👍

Comment (Safirangi) [reply Quiz-App/README.md:28]: hey, I created a gif, but it has a watermark of the editing software. does that work?

Comment (Susaksham) [reply Quiz-App/README.md:28]: > hey, I created a gif, but it has a watermark of the editing software. does that work?

There are couple of software out there that does not create watermark. You can use those

Comment (Safirangi) [reply Quiz-App/README.md:28]: okay

**Review by mudit023 — APPROVED** [review]
LGTM! Thanks for the contribution.

**Pull request merged**